tests/lib/rook/base.py

```diff
@@ -67,18 +69,22 @@ def execute_in_ceph_toolbox(self, command, log_stdout=False):
         return self.kubernetes.execute_in_pod(
             command, self.toolbox_pod, log_stdout=False)
 
+    @abstractmethod
+    def _get_charts(self):
+        pass
+
+    @abstractmethod
+    def _get_helm(self):
+        pass
+
     def install(self):
-        # TODO(jhesketh): We may want to provide ways for tests to override
-        #                 these
+        self.kubernetes.kubectl("create namespace rook-ceph")
+        self._install_operator()
+
         self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'common.yaml'))
+            os.path.join(self.ceph_dir, 'cluster.yaml'))
         self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'operator.yaml'))
-
-        # set operator log level
-        self.kubernetes.kubectl(
-            "-n rook-ceph set env "
-            "deployment/rook-ceph-operator ROOK_LOG_LEVEL=DEBUG")
+            os.path.join(self.ceph_dir, 'toolbox.yaml'))
 
         # reduce wait time to discover devices
         self.kubernetes.kubectl(
```

Comment:
I wonder if we need to be able to wait for the operator to be running before we install the cluster.yaml?

It doesn't seem like it since it is green, but I fear there could be a race here where the CRD's or something isnt' ready.

Reply:
AFAIK this is kubernetes job to handle that properly.

I did not encounter issues with taht in my testing, so I would let it as is.